lambda-natchez/shared/src/main/scala/feral/lambda/natchez/AwsTags.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package feral.lambda
+package feral.lambda.natchez
```

Comment:
Can we add Scalafix migrations for these renames?

Reply:
Added

Reply:
Okay so I've added inputs and outputs but what's the common practice for adding the new v0.3.0 dependency, to run those tests? Do we require a 0.2->0.3 subproject and a 0.3->0.4 subproject?

Reply:
I see the examples in cats have subprojects per version, I will work on that later

Reply:
Instead of more subprojects, I wonder if we can just delete / replace the old migrations with the new ones since we are doing a version bump? I would prefer not to maintain them forever esp. if Scalafix/Scalameta may issue deprecations in the future 🤔 

These migrations are already configured in Scala Steward with a hard-coded version.

https://github.com/scala-steward-org/scala-steward/blob/0765eec5afc158c2e30f5740b4aa23fc7ae0b7dd/modules/core/src/main/resources/scalafix-migrations.conf#L250-L255

Reply:
Removed